deployment/jobs/benchmark/timing-test-job.yaml

```diff
@@ -35,16 +35,18 @@ spec:
             value: "/cache/schema.json"
           - name: RESULTS_PATH
             value: "/tmp/results.json"
-          - name: OBJECT_STORE_ACCESS_KEY
-            valueFrom:
-              secretKeyRef:
-                name: anonlink-benchmark-aws-credentials
-                key: OBJECT_STORE_ACCESS_KEY
-          - name: OBJECT_STORE_SECRET_KEY
-            valueFrom:
-              secretKeyRef:
-                name: anonlink-benchmark-aws-credentials
-                key: OBJECT_STORE_SECRET_KEY
+          - name: OBJECT_STORE_BUCKET
```

Comment:
don't you also have to define `OBJECT_STORE_SERVER`?

Reply:
No we default to assuming S3 is used